tests/attr/test_feature_ablation.py

```diff
@@ -345,15 +349,19 @@ def _multi_input_batch_scalar_ablation_assert(self, func):
 
```

Comment:
Can you also add type hints to the helper functions above, _multi_input_batch_scalar_ablation_assert, _single_input_multi_sample_batch_scalar_ablation_assert, and _single_input_one_sample_batch_scalar_ablation_assert? Thanks!

Reply:
added